libnavigation-core/src/main/java/com/mapbox/navigation/core/history/model/HistoryEventPushHistoryRecord.kt

```diff
@@ -0,0 +1,47 @@
+package com.mapbox.navigation.core.history.model
+
+/**
+ * Allows to read events set with [com.mapbox.navigator.HistoryRecorderHandle.pushHistory]
+ *
+ * @param type corresponding to the event type from [com.mapbox.navigator.HistoryRecorderHandle.pushHistory]
+ * @param properties corresponding to the eventJson from [com.mapbox.navigator.HistoryRecorderHandle.pushHistory]
```

Comment:
should docs be added when we start exposing the `class`
```suggestion
 * @param type 
 * @param properties 
```